apps/miranum-console/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "miranum-console",
-	"displayName": "miranum-console",
+	"displayName": "Miranum: Console",
```

```suggestion
	"displayName": "Miranum: Console",
	"version": "0.1.0",
```
Sonst funktioniert die Extension nicht wenn man sie observed.